### DISCUSSION TITLE
Interesting use of the Transcription AI tool

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [intel/openvino-plugins-ai-audacity#73] [Show and tell]
@RyanMetcalfeInt8 

Hi Ryan

I thought you might be interested in this use of the Whisper transcription tool posted on Facebook on Paul Licameli's Aidacity Voiceover User's Group

A user called Brian Rollins wrote
>This is game-changing. The AI Transcription tool drops the text into a label below the waveform. Makes it WAY easier to find the part of the audio you're looking for.

![426478110_7929449277068697_5652947536922393472_n](https://github.com/intel/openvino-plugins-ai-audacity/assets/50205984/4fd62911-ac92-46cf-b26f-e06e06e85fd3)

Cute, eh?

This is Paul's FB group:
https://www.facebook.com/groups/562223697221079/?hoisted_section_header_type=recently_seen&multi_permalinks=6938056626304389

Peter

**Comment by RyanMetcalfeInt8** [comment]
Hi @petersampsonaudacity,

Thank you for sharing! This use case is exactly how I envisioned this feature being (mostly) used. Even if you don't end up exporting the labels, performing transcription is a fast & easy way to easily find specific audio snippets that you're looking for.

Cheers!
